gui/src/components/mainInput/TipTapEditor/extensions/Prompt/PromptBlockPreview.tsx

```diff
@@ -40,7 +40,7 @@ export const PromptBlockPreview = ({
         inputId={inputId}
         itemId={item.id.itemId}
         onDelete={handleDelete}
-        onTitleClick={handleTitleClick}
+        onTitleClick={!item.content ? undefined : handleTitleClick}
```

Comment:
```suggestion
        onTitleClick={item.content ? handleTitleClick : undefined}
```

(unnecessary negation)